yascheduler.conf

```diff
@@ -0,0 +1,39 @@
+[db]
+user = postgres
+password =
+database = absolidix
+host =localhost
+port = 5432
+
+[local]
+# data_dir = ./data
+# tasks_dir = %(data_dir)s/tasks
+# keys_dir = %(data_dir)s/keys
+engines_dir = %(data_dir)s/engines
+data_dir = /data/yascheduler
+webhook_url = http://localhost:7050/calculations/update?key=bff_api_key
+
+[remote]
+data_dir = ./data
+tasks_dir = %(data_dir)s/tasks
+engines_dir = %(data_dir)s/engines
+
+[clouds]
+hetzner_token = ngzcJHBUZ0MdbjrbrNaVOtDXmE8iXIpGvnTYVPx02wgHXsKxcbMCh2uhnGIbHj8X
```

Comment:
@knopki I just discarded the token, no worries

Reply:
OMG, thanks